### PR TITLE
Update FAQ so users can easily beautify their PowerShell

### PR DESCRIPTION
**Pull request by Dynesshely**
# PR Summary

Update FAQ so users can easily beautify their PowerShell

## PR Context

I always suggest my friends to use PowerShell, then they tried with complain that why their pwsh are not as beautiful as mine, so I added this and next time I can send them only a link.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by Dynesshely** [comment]
> I don't think this is a FAQ (which is more targeted for users of this repo), perhaps we can mention oh-my-posh in https://github.com/PowerShell/PowerShell/tree/master/docs/learning-powershell instead?
> 
> >  GitHub**[PowerShell/docs/learning-powershell at master · PowerShell/PowerShell](https://github.com/PowerShell/PowerShell)**PowerShell for every system! Contribute to PowerShell/PowerShell development by creating an account on GitHub.

That's good, I'll move it to https://github.com/PowerShell/PowerShell/tree/master/docs/learning-powershell

**Comment by iSazonov** [comment]
I guess JanDeDobbeleer should be added to exclusions to pass CI.

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -102,6 +102,15 @@ Note that all bash commands should continue working on PowerShell session.
...
 
+### Make PowerShell Beautiful
+
+You can use [oh-my-posh](https://ohmyposh.dev/) and see
+its source code in [JanDeDobbeleer/oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh).
+
+An Example:
+
+![image](https://user-images.githubusercontent.com/50760269/206097676-58283ed0-d244-4c5c-813d-1218f8bed27f.png)
+
```
Comment (sdwheeler) [inline docs/learning-powershell/README.md:113]: Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?

Also, we already have docs about customizing the shell experience.

Comment (SeeminglyScience) [reply docs/learning-powershell/README.md:113]: > Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?
> 
> Also, we already have docs about customizing the shell experience.

Yeah it feels a little odd to single out oh-my-posh here when there are quite a few projects in the same area (https://github.com/Jaykul/PowerLine, https://github.com/dahlbyk/posh-git, https://github.com/starship/starship).

If the goal is to have a link to send to folks, I'd recommend writing up a markdown gist with your settings and links to projects

Comment (iSazonov) [reply docs/learning-powershell/README.md:113]: > Also, we already have docs about customizing the shell experience.

Has the doc any links to third-party tools?

Comment (Dynesshely) [reply docs/learning-powershell/README.md:113]: > Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?
> 
> Also, we already have docs about customizing the shell experience.

You're right. And where are the docs about customizing the shell experience? Tnx.

**Comment by sdwheeler** [comment]
https://learn.microsoft.com/en-us/powershell/scripting/learn/shell/optimize-shell?view=powershell-7.3

**Comment by Dynesshely** [comment]
So it doesn't look like the changes really helped much, close.

**Pull request closed without merging**